### ISSUE TITLE
More sequences, but fewer ASVs?

### ISSUE THREAD

**Issue by zxl124** [benjjneb/dada2#429]
Hi,

I was running an implementation of dada2 in qiime2 on my dataset of ~120k sequences. With default parameters, only 32k sequences survived the pipeline, and it produced 820 ASVs. I then tried a run with maxEE=5, which resulted in more sequences surviving at the end (71k), but those 71k sequences actually produced less ASVs (557) than the previous run with maxEE=2. Is it common that more sequences could lead to less ASVs?

My thinking is that by changing maxEE to 5, I am allowing reads with more errors, which changes the results of the "learn error rate" step that the model considers more differences in sequences to be errors instead of true differences. This then leads to fewer ASVs. Am I right?

My other question is if changing maxEE can lead to dramatic differences in the number of ASVs, which maxEE should I be using? Was it wrong to change it to 5?

Thank you very much.

Jay

**Comment by joey711** [comment]
I guess the first question is, "How out of date is the version of dada2 in QIIME 2?"... Last summer it was unfortunately quite behind, and they were using some questionable parameter choices...

**Comment by zxl124** [comment]
I am using the 2018.2 version which is the newest version of qiime2. According to the q2-dada2 github, it appears that they are using the 1.6.0 version of dada2.

**Comment by benjjneb** [comment]
> My thinking is that by changing maxEE to 5, I am allowing reads with more errors, which changes the results of the "learn error rate" step that the model considers more differences in sequences to be errors instead of true differences. This then leads to fewer ASVs. Am I right?

That is plausible. When I've looked in the past, there was little effect of varying `maxEE` on the number of output ASVs: https://twitter.com/bejcal/status/771010634074820608

However, over a whole study there might be more effect. I suspect that the difference in "richness" observed in each sample is probably a lot less than over the study as a whole though.

> My other question is if changing maxEE can lead to dramatic differences in the number of ASVs, which maxEE should I be using? Was it wrong to change it to 5?

In general I would lean towards a tighter `maxEE`. In theory, adding more error-ful sequencines shouldn't hurt, but it also should't really help, so I tend to prefer avoiding the possibility of aberrant sequences that might show up at low qualities and mess with the error model.

**Comment by zxl124** [comment]
I reviewed my sequences again, and found that the issue described by me was caused by the presence of primer sequences. I had a step to remove them, but it didn't work properly. I correctly removed primer sequences and tried again. This time dada2 produced the same amount of ASVs regardless of maxEE. I know primer sequences would cause trouble, but didn't know it would have different effects with different maxEE. Anyway, thank you for answering my questions.
